common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2022,8 +2022,20 @@
   },
   "INTERVAL_ARITHMETIC_OVERFLOW" : {
     "message" : [
-      "<message>.<alternative>"
+      "Integer overflow while operating with intervals."
     ],
+    "subClass" : {
+      "WITHOUT_SUGGESTION" : {
+        "message" : [
+          "Try devising appropriate values for the interval parameters."
```

Comment:
Could you clarify what do you mean here.

Reply:
User has integer overflow in make_[dt|ym]_interval and we have no good advise for user. User should take a look into usage of this function and check how he uses it.
I've tried to write some specific text here: https://github.com/apache/spark/pull/48773#discussion_r1831615829
But this error not just for datetime intervals, it's general for all interval integer overflows where we can't help user.

Reply:
@MaxGekk So is current text OK or I should change it?